Helper/CriteriaHelper.php

```diff
@@ -149,4 +151,48 @@ public function generateListBlogPostCriterion( Location $location, array $viewPa
         return new Criterion\LogicalAnd( $criteria );
     }
 
+    public function getSortClauseBySortField( $sortField, $sortOrder = Location::SORT_ORDER_ASC )
```

Comment:
@andrerom Besides making getSortClauseBySortField public in LocationService, can you help me understand what you are suggesting for the type hinting? Also why couldn't UserService and the others also use getSortClauseBySortField if exposed as public instead of repeating ourselves?